body.tex

```diff
@@ -228,7 +235,7 @@ \section{Short Term Planning}
 \label{sec:cycle-plan}
 
 Short term planning is carried out in blocks referred to as \glspl{cycle}, which (usually) last for six months.
-Before the start of a \gls{cycle}, milestones should be confirmed by the Director's Office.
+Before the start of a \gls{cycle}, milestones are confirmed by the Director's Office, listed in Smartsheet, and detailed in the Milestone issue.
```

Comment:
I would add "Any team member can find the milestones in Jira." (RIght?)

Reply:
agreed.  i'll add this